src/Runner.Common/ResultsServer.cs

```diff
@@ -0,0 +1,103 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using GitHub.DistributedTask.WebApi;
+using GitHub.Services.Results.Client;
+
+namespace GitHub.Runner.Common
+{
+    [ServiceLocator(Default = typeof(ResultServer))]
+    public interface IResultsServer : IRunnerService, IAsyncDisposable
```

Comment:
do you need `IAsyncDisposable`?

Reply:
I was going to move the web socket client to here in the future so I left this.  I will remove this for now since it's not needed.